embeddings/pipeline/flair_pair_classification.py

```diff
@@ -60,11 +65,24 @@ def __init__(
             transformation = transformation.then(
                 SampleSplitsFlairCorpusTransformation(*sample_missing_splits, seed=seed)
             )
-        embedding = AutoFlairDocumentPoolEmbedding.from_hub(
-            repo_id=embedding_name,
-            document_embedding_cls=document_embedding_cls,
-            **load_model_kwargs if load_model_kwargs else {}
-        )
+
+        if isinstance(embedding_name, pathlib.Path):
+            if not model_type_reference:
+                _logger.error("For embedding loaded directly from file model_type_reference must be provided!")
+
+            embedding = AutoFlairDocumentPoolEmbedding.from_file(
+                file_path=embedding_name,
+                model_type_reference=model_type_reference,
+                document_embedding_cls=document_embedding_cls,
+                **load_model_kwargs if load_model_kwargs else {}
+            )
+        else:
+            embedding = AutoFlairDocumentPoolEmbedding.from_hub(
+                repo_id=embedding_name,
+                document_embedding_cls=document_embedding_cls,
+                **load_model_kwargs if load_model_kwargs else {}
+            )
+
```

Comment:
I see that it is repeated as in `flair_classification`. Maybe it should be extracted to a method and incorporated in the `StandardPipeline`?

Reply:
Currently skipped. Could be addressed in future.